user_data/strategies/notifier.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import sys
+sys.path.append('/root/workspace2/execution/')
```

Comment:
do not import from other repo

Reply:
This is public repo so I didn't want to expose our token and channel ID that's why but if you still want me to do it that way I will

Reply:
Good point. OK - Use Config. EXECUTION_PATH